### PR TITLE
JP Onboarding: round the corners on the user type step

### PR DESCRIPTION
**Pull request by keoshi**
#### Changes proposed in this Pull Request

* Round the corners on the user type step of Jetpack onboarding.

#### Testing instructions

* Spin up this PR.
* Go to `/jetpack/connect/user-type/SITE_URL`.
* Ensure the first and last options (the only two for now) have rounded corners that match the site type and site topic steps.

#### Before

image

#### After

image

### CODE REVIEW

File: client/jetpack-connect/user-type/style.scss [hunk 1]
```diff
@@ -50,6 +50,15 @@
 	&:focus {
 		box-shadow: inset 0 0 0 2px var( --color-accent );
 	}
+
+	&:first-of-type {
```
Comment (tyxla) [inline client/jetpack-connect/user-type/style.scss]: Could we use `:first-child` and `:last-child` instead, for consistency reasons? Seems like that's what we're already using for the site type component:

https://github.com/Automattic/wp-calypso/blob/e64001e08c5c7d24be02021b4609e171a84ab2ae/client/signup/steps/site-type/style.scss#L31

Comment (keoshi) [reply client/jetpack-connect/user-type/style.scss]: Good catch! I've updated it on https://github.com/Automattic/wp-calypso/pull/31712/commits/536284fb5a4246a60380ed92999d0c4cc841c1d2

**Comment by keoshi** [comment]
Folletto has given this the 👍in Slack — merging now as it's a small CSS change.

**Comment by jeffgolenski** [comment]
Thanks Filipe!

**Pull request merged**